packages/vitaminjs/bin/vitamin-jest.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable global-require */
+try {
+    require('vitaminjs-jest/bin/vitamin-jest');
+} catch (e) {
+    if (e.code !== 'MODULE_NOT_FOUND') {
```

Comment:
We may output a message to explain how to install `vitamin-jest` instead of catching and do nothing. By the way `MODULE_NOT_FOUND` error can also be thrown by the app itself. We should check if the required module was `vitamin-jest`

Reply:
should not happen